.vscode/launch.json

```diff
@@ -338,4 +338,4 @@
             ]
         }
     ]
-}
+}
```

Comment:
Let's remove such things from commits. Otherwise, it is hard to navigate any significant changes in file history.